.github/workflows/tests.yml

```diff
@@ -20,6 +20,7 @@ jobs:
         php-version: [ "8.0" , "8.1", "8.2" ]
         operating-system:
           - "ubuntu-latest"
+#          - "windows-latest" # TODO: Windows is not working in the CI at the moment for some reason
```

Comment:
I downloaded and ran the tests using a Windows machine, all tests are green now.

![phpunit-gacela-windows-error](https://github.com/gacela-project/gacela/assets/6381924/1c8c1ef0-e090-444d-bc2e-243ce438cd5c)

But the `GitHub Actions` is failing, in any case, I think this commit brings value to Windows users, and we can investigate in the future why the CI for Windows is not working. Also, I am unable to reproduce this error 🤔 

![image](https://github.com/gacela-project/gacela/assets/6381924/0f2a29b3-b454-4eb3-a1ad-34ff4b67b8b2)

Trace: https://github.com/gacela-project/gacela/actions/runs/5171780050/jobs/9315512655

Reply:
Yes, let's merge it with your changes -as they provide value as it is-, and we can continue the investigation about why is the Windows workflow not working properly as they are in your windows machine.